src/mame/drivers/p2000t.cpp

```diff
@@ -97,7 +176,7 @@ GFXDECODE_END
 
 /* Keyboard input */
 
-/* 2008-05 FP:
+/* 2008-05 FP:   <--- 2020-12-16 Bekkie: Confirmed these postions are correct --->
```

Comment:
What do you mean by this?  If the positions are confirmed and the code is correct, the TODO can be removed.